docs/documentation/maya/skin.md

```diff
@@ -1,127 +1,185 @@
-| Soft constraints  | Float | 0.0   | \[0.0, 1.0\]  | Weight to modulate the correction applied to the vertices to keep them at a constant distance to the closest point on the reference mesh at initialization. <br> This attribute is paintable and normalized together with *Slide Constraints* and *Hard Constraints*. |
+ - **Global Mass Multiplier** (Float, 1.0): Sets the scaling factor applied to the mass of every point.
+    - Has a range of \[0.0, 10.0\] [^1]
+ - **Global Damping** (Float, 0.75): Sets the scaling factor applied to the global damping of every point.
+    - Has a range of \[0.0, 1.0\] [^1]
+ - **Inertia Damper** (Float, 0.0): Sets the linear damping applied to the dynamics of every point.
+    - Has a range of \[0.0, 1.0\] [^1]
+ - **Rest Length Multiplier** (Float, 1.0): Sets the scaling factor applied to the edge lengths at rest.
+    - Has a range of \[0.0, 2.0\] [^1]
+ - **Max Sliding Distance** (Float, 0.5): Determines the size of the sliding area. It corresponds to the maximum distance to the closest point on the reference mesh computed on initialization.
+    - The higher this value is, the higher quality and the lower performance.
+    - Has a range of \[0.0, 10.0\] [^1]
+ - **Compression Multiplier** (Float, 1.0): Sets the scaling factor applied to the compression resistance of every point.
+    - Has a range of \[0.0, 2.0\] [^1]
+ - **Stretching Multiplier** (Float, 1.0): Sets the scaling factor applied to the stretching resistance of every point.
+    - Has a range of \[0.0, 2.0\] [^1]
+ - **Attenuation Velocity Factor** (Float, 1.0): Sets the weight of the attenuation applied to the whole simulation driven by the Attenuation Matrix.
+    - Has a range of \[0.0, 10.0\] [^1]
+ - **Sliding Constraints Mode** (Enumerator, "Fast"): Defines the mode of execution for the sliding constraints.
+    - *Quality* is more accurate, recommended for final results.
+    - *Fast* provides higher performance, recommended for preview.
 
 ## Attribute Editor Template
 
 <figure markdown>
   ![skin editor first part](images/attribute_editor_part_one_skin.png) 
-  <figcaption>Figure 1: Skin Attribute Editor</figcaption>
+  <figcaption><b>Figure 2</b>: AdnSkin Attribute Editor</figcaption>
 </figure>
 
 <figure markdown>
   ![skin editor second part](images/attribute_editor_part_two_skin.png)
-  <figcaption>Figure 2: Skin Attribute Editor (Advanced Settings)</figcaption>
+  <figcaption><b>Figure 3</b>: AdnSkin Attribute Editor (Advanced Settings)</figcaption>
+</figure>
+
+## Debugger
+
+In order to better visualize deformer constraints and attributes in the Maya viewport there is the option to enable the debugger, found in the dropdown menu labeled *Debug* in the attribute editor.
```

Comment:
```suggestion
In order to better visualize deformer constraints and attributes in the Maya viewport there is the option to enable the debugger, found in the dropdown menu labeled *Debug* in the Attribute Editor.
```

Reply:
After looking at the rendered page, we have some not standarized the references to other attributes. Sometimes we use Italic, sometimes we use plain format. I would go with Italic

Reply:
Change applied to the whole document!

Reply:
This is not applied

Reply:
Change applied now!